### PR TITLE
Add comparison and reset capability to novice module

### PR DESCRIPTION
**Pull request by stefanv**
Partially addresses #1588 and #1589

### CODE REVIEW

File: skimage/io/_io.py [hunk 1]
```diff
@@ -156,6 +157,26 @@ def imshow(arr, plugin=None, **plugin_args):
     return call_plugin('imshow', arr, plugin=plugin, **plugin_args)
 
 
+def imshow_collection(ic, plugin=None, **plugin_args):
+    """Display an image.
```
Comment (ahojnnes) [inline skimage/io/_io.py]: "Display a collection of images." ?

**Comment by ahojnnes** [comment]
LGTM except for the Travis failure.

**Comment by stefanv** [comment]
I'm afraid my matplotlib foo is not enough to solve this one.  Any ideas?

**Comment by blink1073** [comment]
@stefanv, mind rebasing this to trigger another build?

**Comment by stefanv** [comment]
Let's see!

**Comment by stefanv** [comment]
Funny result from Travis!

**Comment by blink1073** [comment]
Maybe try `assert_almost_equal`?

**Comment by stefanv** [comment]
@blink1073 In this case, the object defines its own ``__eq__`` operator, and we're only working with integers.  I have no idea why it fails!

**Comment by blink1073** [comment]
Are we *sure* they're integers?

**Comment by stefanv** [comment]
We're good to go on this one!  Failure is for Coverage only.

Pinging @scikit-image/core to review for 0.12.

**Comment by JDWarner** [comment]
:+1: LGTM

**Comment by blink1073** [comment]
LGTM, thanks @stefanv!

**Pull request merged**